pkg/dmesg/watcher.go

```diff
@@ -221,3 +241,34 @@ func findISOTimestampIndex(s string) int {
 	}
 	return loc[0]
 }
+
+func (l LogLine) cacheKey() string {
+	h := sha256.New()
```

Comment:
Do we need to use sha256 here?  Perhaps we should a simple hash function like fnv-64 or the simplest way is use fmt.Sprintf("%d-%s", l.Timestamp.Unix(), l.Content)

Reply:
Actually go-cache already hash map... let me just raw string https://github.com/patrickmn/go-cache/blob/46f407853014144407b6c2ec7ccc76bf67958d93/cache.go#L42